crates/weaver_semconv/src/attribute.rs

```diff
@@ -410,6 +410,14 @@ pub enum Examples {
     Bools(Vec<bool>),
     /// A array of strings example.
     Strings(Vec<String>),
+    /// List of arrays of integers example.
+    ListOfInts(Vec<Vec<i64>>),
```

Comment:
Could you add a test to ensure this is working?

Reply:
I've added tests... feels like I'm just testing serde_yaml though ;)

Reply:
Thanks, and agreed. I had intended for those tests to be on full resolution ensuring these are preserved through the resolution process, but what you have is fine for now.